src/handlers/localstorage/defiPulse.ts

```diff
@@ -0,0 +1,17 @@
+import { getGlobal, saveGlobal } from './common';
+import { IndexToken } from '@rainbow-me/entities';
+
+// Key used for loading the cache with data from storage
+export const DEFI_PULSE_FROM_STORAGE = 'defiPulseFromStorage';
+
+const DEFI_PULSE = 'defiPulse';
+
+interface DpiResult {
+  base: IndexToken;
+  underlying: IndexToken[];
+}
+
+export const getDefiPulse = (): Promise<DpiResult> => getGlobal(DEFI_PULSE, []);
```

Comment:
`getDefiPulse` is not called from anywhere (we should add it to `useLoadGlobalData` similar to the walletBalances prefetch)

Also the default empty state should be null instead of []

Reply:
I'm deleting cause I don't wanna get this from localstorage anymore

Reply:
I'm deleting it because we don't need it anymore.